gameOOP.js

```diff
@@ -535,22 +540,22 @@ class Game {
                 subtype: "transition",//for level transition, portals, trapdoors, stairs, holes, etc
             },
             "chest": {
-                id: 10,
+                id: 11,

```

Comment:
we should not have duplicates now.
we can also base the id on the index of the array instead